### ISSUE TITLE
Grammatical error in Burger section in documentation

### ISSUE THREAD

**Issue by Asipita** [mantinedev/mantine#4442]
### Link to the page where something nasty is located

https://mantine.dev/core/burger/

### Exact quote of what is wrong

Burger renders a regular button element. Include `title` or `aria-label` props for screen reader support as by default element does not associated label.

### Are you willing to create pull request with the fix?

Yes

**Comment by rtivital** [comment]
you are welcome to submit a PR with a fix

**Comment by Asipita** [comment]
Done.

https://github.com/mantinedev/mantine/pull/4444
